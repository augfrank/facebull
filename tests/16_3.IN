M0   C0   C2  2246
M1   C0   C3  2384
M2   C0   C4  2470
M3   C0   C6  2184
M4   C0   C10  2961
M5   C0   C11  2545
M6   C0   C13  2788
M7   C1   C7  2537
M8   C1   C12  2023
M9   C1   C14  2580
M10   C2   C4  2200
M11   C2   C5  2634
M12   C2   C6  2908
M13   C2   C7  2041
M14   C2   C12  2930
M15   C2   C13  2945
M16   C2   C15  2735
M17   C3   C8  2015
M18   C3   C9  2304
M19   C3   C15  2664
M20   C4   C1  2446
M21   C4   C3  2689
M22   C4   C6  2108
M23   C4   C9  2234
M24   C4   C12  2751
M25   C5   C2  2440
M26   C5   C4  2980
M27   C5   C7  2726
M28   C5   C8  2038
M29   C5   C9  2052
M30   C5   C10  2469
M31   C5   C11  2106
M32   C5   C15  2670
M33   C6   C1  2445
M34   C6   C3  2903
M35   C6   C4  2894
M36   C6   C8  2293
M37   C6   C12  2381
M38   C7   C2  2536
M39   C7   C8  2497
M40   C7   C9  2597
M41   C7   C13  2641
M42   C8   C3  2156
M43   C8   C5  2689
M44   C8   C9  2809
M45   C8   C12  2002
M46   C8   C14  2678
M47   C8   C15  2633
M48   C9   C0  2393
M49   C9   C2  2836
M50   C9   C5  2023
M51   C9   C11  2750
M52   C9   C13  2790
M53   C9   C15  2262
M54   C10   C1  2068
M55   C10   C2  2594
M56   C10   C4  2107
M57   C10   C5  2856
M58   C10   C6  2710
M59   C10   C8  2998
M60   C10   C9  2994
M61   C10   C13  2904
M62   C10   C14  2940
M63   C11   C2  2231
M64   C11   C3  2001
M65   C11   C14  2968
M66   C12   C4  2976
M67   C12   C6  2968
M68   C12   C7  2334
M69   C12   C9  2976
M70   C12   C10  2098
M71   C12   C14  2326
M72   C12   C15  2912
M73   C13   C0  2173
M74   C13   C2  2800
M75   C13   C5  2781
M76   C13   C8  2822
M77   C13   C11  2165
M78   C14   C3  2489
M79   C14   C4  2648
M80   C14   C5  2398
M81   C14   C6  2561
M82   C14   C12  2384
M83   C14   C13  2301
M84   C14   C15  2795
M85   C15   C0  2401
M86   C15   C8  2390
M87   C15   C9  2485
M88   C15   C11  2274
M89   C15   C14  2965
