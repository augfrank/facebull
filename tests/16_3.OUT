35562
0 9 13 17 22 29 33 41 45 53 56 64 70 77 80 85
